### PR TITLE
Add string into radar.lua and template FR

### PR DESCRIPTION
**Pull request by d-stephane**
- One area unit (@1m) up/down\n-> no claims on this Y level
- square
- area
- nodes

### CODE REVIEW

**Comment by SmallJoker** [comment]
Your master branch is outdated. Please rebase it -> `git rebase upstream/master`
EDIT: Also consider to adjust your git client's settings. The email address doesn't match with the one you use on GitHub.

File: radar.lua [hunk 1]
```diff
@@ -104,10 +104,10 @@ s_protect.command_radar = function(name)
 			minetest.formspec_escape("[combine:300x300"
 				.. parts .. marker_str)
 				.. dir_mod .. "]" ..
-		"label[0,6.8;1 square = 1 area = "
+		"label[0,6.8;1 " .. S("square") .. " = 1 " .. S("area") .. " = "
```
Comment (SmallJoker) [inline radar.lua]: You could simply translate the entire thing.
`S("1 square = 1 area = @1x@2x@3 nodes (X,Y,Z)")` (or similar)

File: radar.lua [hunk 2]
```diff
@@ -121,7 +121,6 @@ s_protect.command_radar = function(name)
 		"label[7,4.25;" .. S("Access for everybody") .. "]" ..
 		"image[6,5;1,1;simple_protection_radar_down.png]" ..
 		"image[7,5;1,1;simple_protection_radar_up.png]" ..
-		"label[6,6;One area unit ("..s_protect.claim_height
-			.. "m) up/down\n= no claims on this Y level]"
+		"label[6,6;" .. S("One area unit (@1m) up/down\n-> no claims on this Y level", s_protect.claim_height) .. "]"
```
Comment (SmallJoker) [inline radar.lua]: Line too long. Please break it after `",`

File: radar.lua (hunk 1 above)
Comment (d-stephane) [reply radar.lua]: Good idea, modif ok in my repository

File: radar.lua (hunk 2 above)
Comment (d-stephane) [reply radar.lua]: line break : ok

**Comment by SmallJoker** [comment]
@d-stephane Is the PR ready for merge (as in, grammar corrections)? I've tested the functionality and it works as wanted.

**Comment by d-stephane** [comment]
It's ok for me

**Pull request merged**